config/rewrite.yml

```diff
@@ -5,9 +5,9 @@ displayName: Auto Fixes
 description: List of auto fixes from different providers.
 recipeList:
   - org.checkstyle.autofix.CheckstyleAutoFix:
-      violationReportPath: "target/cs_errors.xml"
```

Comment:
less is more, assuming we can reduce to final version ->

Reply:
Revert all unrelated changes